app.json

```diff
@@ -1,5 +1,13 @@
 {
   "expo": {
-    "sdkVersion": "15.0.0"
+    "sdkVersion": "15.0.0",
+    "name": "Serlo-ABC",
```

Comment:
If I read the commit messages correctly, then this is used for the production build? Didn't look into how to do that yet. So if possible, could you add a `npm` task for that?